dev/bots/codesign.dart

```diff
@@ -22,11 +21,12 @@ bool isBinary(String filePath) {
   return (result.stdout as String).contains('application/x-mach-binary');
 }
 
-List<String> findBinaryPaths() {
+/// Find every binary file in the given [rootDirectory]
+List<String> findBinaryPaths([String rootDirectory]) {
   final ProcessResult result = Process.runSync(
     'find',
     <String>[
-      cacheDirectory,
+      rootDirectory,
```

Comment:
Shouldn't we default this to something non-null?  Or make it a non-optional parameter and assert it's not null?

Reply:
good catch. i had a fallback default, forget why I deleted it...

Reply:
done